### PR TITLE
fix compilation

### PR DESCRIPTION
**Pull request by AlwinEsch**
There was still the old include way, this changes to the new

### CODE REVIEW

**Comment by akva2** [comment]
Just a heads up. This and the rest with old headers are more than likely not working work in progress things. Efforts more than welcome but be aware should things not behave.

**Comment by AlwinEsch** [comment]
Also was the link against opengl lib missing. After them it was usable without crash, but it destroy a bit the OpenGL Window ;)

bildschirmfoto vom 2016-12-15 23-31-43
bildschirmfoto vom 2016-12-15 23-31-51

But good to have it compiled.

**Comment by AlwinEsch** [comment]
Test of them comes from my add-on system rework and want to have the work for the owners so small as possible.

There comes a lot of changes in future and to always fix after a change can be a little frustrating of the owners, if not direct known whats the reason of them. :)

Here visible what comes: https://github.com/AlwinEsch/screensaver.asteroids/blob/use-cpp-interface/src/main.cpp
There becomes the old way complete changed to `cpp` classes.

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR})
 enable_language(CXX)
 
 find_package(Kodi REQUIRED)
+find_package(OpenGL REQUIRED)
 
 include_directories(${KODI_INCLUDE_DIR})
```
Comment (notspiff) [inline CMakeLists.txt:12]: add include dirs as well.

**Pull request merged**